File: huanletao-manager-web/src/main/webapp/js/service/brandService.js
app.service("brandService",function($http){
    this.findAll = function(){
       return $http.get("../findAll.do");
    }





    this.save = function (url,entity) {
        return $http.post(url,entity);

    }
    this.findOne = function(id){
       return  $http.post("../findOne.do?id="+id)
    }




    this.dele = function(selectIds){
        return $http.post("../dele.do?ids="+selectIds)
    }

    this.search = function(pageNum,pageSize,keywords){
        return $http.get("../findAll.do?pageNum="+pageNum+"&pageSize="+pageSize+"&keywords="+keywords);
    }

    this.selectBrandList = function(){
        return $http.get("../selectBrandList.do");
    }

})

File: huanletao-sellergoods-interface/src/main/java/com/huanletao/sellergoods/service/BrandService.java
package com.huanletao.sellergoods.service;

import com.huanletao.pojo.PageResult;
import com.huanletao.pojo.TbBrand;

import java.util.List;
import java.util.Map;

public interface BrandService {
    PageResult findAll(String keywords, int pageNum, int pageSize);

    void add(TbBrand brand)throws Exception;

    TbBrand findOne(Long id);

    void update(TbBrand brand);

    void dele(Long[] ids);

    List<Map> selectBrandList();
}
